src/test/java/com/iota/iri/service/tipselection/impl/CumulativeWeightCalculatorTest.java

```diff
@@ -269,6 +283,7 @@ public void testTangleWithCircle2() throws Exception {
     }
 
     @Test
+    @Ignore
```

Comment:
It tests a broken thing in our previous system, collisions between HashIds of hashes. 
`transactionHash2` is a similar hash to the other one, and is therefore not included.
No idea why we ever did that, as it is literally skipping on transactions this way.

Reply:
Actually, id vote for removing the hashId entirely, having it might speed some things up, but if it blocks certain tx, its still not worth it. (in the test, tx1 and tx2 work because they reference to the same value in the end.)

An example on how to break it earlier is make tx3 and tx1 the same instead of 1 and 2. Then 3 should be value 1, and not 2, which will error